### PR TITLE
allow listed runners

### PR DESCRIPTION
**Pull request by KevDevSha**
# What does this PR do?

Updating to allow listed runners

### CODE REVIEW

File: .github/workflows/pr-cpu.yaml [hunk 1]
```diff
@@ -9,36 +9,39 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' && github.ref != 'refs/heads/dev' }}
 jobs:
   pytest-cpu:
-    uses: mosaicml/ci-testing/.github/workflows/pytest-cpu.yaml@v0.0.9
+    name: ${{ matrix.name }}
+    runs-on: linux-ubuntu-latest
+    if: github.repository_owner == 'mosaicml'
```
Comment (eitanturok) [inline .github/workflows/pr-cpu.yaml:14]: Do we need this if statement here
```
if: github.repository_owner == 'mosaicml'
```

**Review by eitanturok — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-cpu.yaml (hunk 1 above)
Comment (mvpatel2000) [reply .github/workflows/pr-cpu.yaml:14]: Yes, this is so forks don't run this code as well.

**Review by mvpatel2000 — APPROVED** [review]

**Comment by DinnerMonster** [comment]
Will update merge once we get our emu composer repo cofigured for the container pull

**Comment by KevDevSha** [comment]
Will update and merge once our EMU gh registry for composer is created to be used for the job container.

File: .github/workflows/pr-gpu.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: PR GPU tests
 on:
-  pull_request_target:
+  pull_request:
```
Comment (KevDevSha) [inline .github/workflows/pr-gpu.yaml:3]: change to pull_request_target before merge

**Pull request merged**